### ISSUE TITLE
Supports 0 as a valid number for listing limits

### ISSUE THREAD

**Issue by mniewrzal** [storj/uplink#62]
Amazon S3 supports 0 as a valid number for listing limits. In such a case, it returns an empty result with IsTruncated = false. We should consider the same behavior for compatibility.

**Comment by egonelbre** [comment]
Isn't this compatibility more of a gateway concern?

**Comment by mniewrzal** [comment]
Yeah, that makes sense. I was moving things from our old Metainfo todo confluence list.

@wthorp what do you think about it?
